docker/dev/docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
       - lemmy
 
   postgres:
-    image: postgres:12-alpine
+    image: postgres:14-alpine
```

Comment:
Change this in federation and prod docker-compose files too? Not sure about the latter.